CONTRIBUTING.md

```diff
@@ -41,6 +46,16 @@ As with other types of contributions, the first step is to [**open an issue on G
 
 Once you've opened an issue, check out our [Developer Guide](./DEVELOPER_GUIDE.md) for instructions on how to get started.
 
+_Note_: Since you are a beginner, we advise you to start with simple issues.
```

Comment:
Not all readers of this document are beginners. We should rephrase to be something like, "The easiest way to get started is with simple issues".